core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -3816,8 +3816,6 @@ class KafkaApis(val requestChannel: RequestChannel,
       requestHelper.sendMaybeThrottle(request, shareGroupHeartbeatRequest.getErrorResponse(Errors.GROUP_AUTHORIZATION_FAILED.exception))
       CompletableFuture.completedFuture[Unit](())
     } else {
-      /* TODO: Once ShareGroupHeartBeatAPI is supported in group coordinator, uncomment the code below
-          (lines 3821 - 3833) and remove the part sending INVALID_CONFIG exception (lines 3835, 3836)
```

Comment:
Do we need this log line `info(s"*** SHARE GROUP HEARTBEAT RESPONSE *** + ${response}")` on every reponse in info mode? Maybe debug?

Reply:
my bad, I was using it during my testing. Will remove it now.